src/Lidarr.Api.V1/Albums/AlbumReleaseResource.cs

```diff
@@ -7,15 +7,31 @@ namespace Lidarr.Api.V1.Albums
 {
     public class AlbumReleaseResource
     {
-        public string Id { get; set; }
+        public int Id { get; set; }
+        public int AlbumId { get; set; }
+        public string ForeignReleaseId { get; set; }
         public string Title { get; set; }
-        public DateTime? ReleaseDate { get; set; }
```

Comment:
ex. <https://musicbrainz.org/release-group/1346c819-d57b-4c38-aa55-1e599dfc3645>

Reply:
On `develop` we only have release dates for the release group and not the release, which I have kept in this PR, so I removed the field from the `AlbumReleaseResource` because it's not used and it would just be returning the date from the `Album` anyway.  I can update the metadata to fetch the release date for each `AlbumRelease` if you want.

Reply:
I agree. I'll have a go at sorting this out. Might rename the the Album release date as OriginalYear since that's what it is